### PR TITLE
Formatting updates using newest version of clang-format

### PR DESCRIPTION
**Pull request by oddbookworm**

### CODE REVIEW

File: src_c/freetype/ft_wrap.c [hunk 1]
```diff
@@ -38,8 +38,7 @@ ft_wrap_quit(pgFontObject *);
  *
  *********************************************************/
 void
-_PGFT_SetError(FreeTypeInstance *ft, const char *error_msg, FT_Error error_id)
-{
+_PGFT_SetError(FreeTypeInstance *ft, const char *error_msg, FT_Error error_id){
```
Comment (ankith26) [inline src_c/freetype/ft_wrap.c:41]: The changes in this file are weird... but we will just go with whatever clang-format is doing here

**Review by ankith26 — APPROVED** [review]
LGTM, thanks for the PR

**Comment by ankith26** [comment]
There seems to be a new mypy fail too, as it catches a real issue. Maybe that can be handled in this PR as well, just gotta add something like
```py
def colorspace(
    surf: Surface, color: Literal["YUV", "HSV"], dest_surf: Surface = ..., /
) -> Surface: ...
```
to `camera.pyi`

**Comment by oddbookworm** [comment]
> There seems to be a new mypy fail too, as it catches a real issue. Maybe that can be handled in this PR as well, just gotta add something like
> 
> ```python
> def colorspace(
>     surf: Surface, color: Literal["YUV", "HSV"], dest_surf: Surface = ..., /
> ) -> Surface: ...
> ```
> 
> to `camera.pyi`

Updated. I also updated the doc signature to match

**Review by novialriptide — APPROVED** [review]

**Pull request merged**